Tribler/Core/Modules/restapi/events_endpoint.py

```diff
@@ -88,9 +121,9 @@ def render_GET(self, request):
         """
         self.events_request = request
 
-        request.write(json.dumps({"type": "events_start"}))
+        request.write(json.dumps({"type": "events_start"}) + '\n')
```

Comment:
No, that's a boundary between the different events. In the GUI, I split on the newline to get each event. If I don't have a separator between the JSON dictionaries, parsing will be a lot harder.